### DISCUSSION TITLE
torch_geometric.nn.models for classification

### DISCUSSION THREAD

**Discussion by Takaogahara** [pyg-team/pytorch_geometric#5038] [Q&A] [answered]
Hello everyone,

I've been looking in the documentation and Github issues for some time now and haven't been able to find a way to use the models available in `torch_geometric.nn` such as [GCN](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.models.GCN) for graph classification.

Is there any way to use them or are they just for node classification?

Thanks!

**Comment by rusty1s** [comment, marked as answer]
You can use the models in `nn.models` as part of graph classification pipeline:
```python
x = self.gcn(x, edge_index)
x = global_mean_pool(x, batch)
return self.classifier(x)
```
We assume that a GNN just learns new node representations, which can then be used for various down-stream tasks (node, edge, graph) by connecting different heads to it.

**Reply by Takaogahara** [reply]
Thanks!!!
